src/server/origin/origin-type/index.test.js

```diff
@@ -73,4 +74,10 @@ describe('OriginTypePage', () => {
     expect(plugin.name).toBe(`${sectionKey}-${questionKey}`)
     expect(plugin).toHaveProperty('register')
   })
+
+  describePageSnapshot(
```

Comment:
@DrogoNevets Just FYI, once this is merged we'll have a new helper for snapshot tests.